pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -28,6 +28,7 @@ package queue
 
 import (
 	"fmt"
+	"k8s.io/kubernetes/pkg/features"
```

Comment:
nit: Move it into the next section (along with L41).

Reply:
fixed and reviewed. resolving